src/index.js

```diff
@@ -47,6 +78,15 @@ class TracePlugin {
       'post:setup': this.postSetup.bind(this),
       'pre:report': this.preReport.bind(this)
     };
+    if (this.config.autoHttp.enabled) {
+      this.autoData = {
```

Comment:
Shouldn't this be `auto Http`? It is likely we ad auto tracing for things other than HTTP, making it clear what's being traced here will be helpful in the long run.

Reply:
Yes let's update

Reply:
Yes